Frontend/ui-customer/src/Components/RateCard/View.purs

```diff
@@ -83,147 +92,361 @@ view push config =
            , imageWithFallback if config.nightCharges then "ny_ic_night,https://assets.juspay.in/nammayatri/images/user/ny_ic_night.png" else "ny_ic_day,https://assets.juspay.in/nammayatri/images/user/ny_ic_day.png"
            ]  
          ]
-      , linearLayout
+      ,linearLayout
         [ width MATCH_PARENT
         , height WRAP_CONTENT
         , orientation HORIZONTAL
-        , padding (Padding 20 20 20 8)
-        ][ textView
-           [ width WRAP_CONTENT
-           , height WRAP_CONTENT
-           , textSize FontSize.a_16
-           , color Color.black800
-           , text $ (getString MIN_FARE_UPTO) <> if config.nightCharges then " 🌙" else ""
-           ]
-         , textView
-           [ width WRAP_CONTENT
-           , height WRAP_CONTENT
-           , textSize FontSize.a_16
-           , color Color.black800
-           , text config.baseFare
-           , gravity RIGHT
-           , weight 1.0
-           ]
-         ]
+        ][PrestoAnim.animationSet [ if config.currentRateCardType == DefaultRateCard then (translateInXBackwardAnim if config.onFirstPage == true then true else false) else (translateInXForwardAnim true) ] $
```

Comment:
`config.onFirstPage` this you can directly assign this value no need of this `if config.onFirstPage == true then true else false`